streetcrm/admin.py

```diff
@@ -638,15 +655,18 @@ def export_search(request, form, advanced, search_query):
         
         last_header=[]
         header=[]
+        
+        search_title = 'Created: {} - Query: {}'.format(
```

Comment:
We might change the exact wording here, but that's TBD.  Obviously, it's a trivial change.